packages/eslint-config-next/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "eslint-config-next",
+  "version": "10.1.3",
+  "description": "ESLint configuration used by NextJS.",
+  "main": "index.js",
+  "license": "MIT",
+  "repository": {
+    "url": "vercel/next.js",
+    "directory": "packages/eslint-config-next"
+  },
+  "dependencies": {
+    "@rushstack/eslint-patch": "^1.0.6",
+    "@babel/core": "^7.13.14",
```

Comment:
I'm trying to understand where `@babel/core` and `@babel/eslint-parser` parser is used, this will increase install size and run a separate version of Babel than the one that Next.js has bundled which will lead to differences in parsing long-term. We'll likely want to expose a Next.js specific eslint parser.

Reply:
Yeah it'll increase install size unfortunately, but I've set it up to point to `next/babel` so that it stays in line with how Babel is configured in Next.js instead of relying on a user's config.

<pre>
  <code>
  parserOptions: {
    <strong>requireConfigFile: false,
    babelOptions: {
      presets: ['next/babel'],</strong>
    },
  },
  </code>
</pre>

> Run a separate version of Babel than the one that Next.js has bundled

Would it make sense to lift `@babel/core` to the root `package.json` in order to share the same version across both workspaces?

> We'll likely want to expose a Next.js specific eslint parser.

Do you mean create a separate, custom eslint parser to handle experimental syntax instead of relying on `@babel/eslint-parser`? Wouldn't we still need to align with Babel parsing (or are you thinking that a custom parser could do the same with minimal delta to install size)?

Reply:
> Do you mean create a separate, custom eslint parser to handle experimental syntax instead of relying on @babel/eslint-parser? Wouldn't we still need to align with Babel parsing (or are you thinking that a custom parser could do the same with minimal delta to install size)?

Bundling the eslint parser in the same way we bundle Babel with ncc, that'll allow for the parser to use the internal bundled Babel version.

Reply:
[Done](https://github.com/vercel/next.js/pull/23702/commits/fbac46ee929bbc8a8377a6f1b1e3cc2a83135eac)!

Reply:
Ah makes sense, thanks.

[Done](https://github.com/vercel/next.js/pull/23702/commits/fbac46ee929bbc8a8377a6f1b1e3cc2a83135eac).